app/app/settings.py

```diff
@@ -532,22 +534,37 @@
 GITHUB_API_TOKEN = env('GITHUB_API_TOKEN', default='')  # TODO
 GITHUB_APP_NAME = env('GITHUB_APP_NAME', default='gitcoin-local')
 
+# Twitter Integration
+TWITTER_CONSUMER_KEY = env('TWITTER_CONSUMER_KEY', default='')  # TODO
+TWITTER_CONSUMER_SECRET = env('TWITTER_CONSUMER_SECRET', default='')  # TODO
+TWITTER_ACCESS_TOKEN = env('TWITTER_ACCESS_TOKEN', default='')  # TODO
+TWITTER_ACCESS_SECRET = env('TWITTER_ACCESS_SECRET', default='')  # TODO
+TWITTER_USERNAME = env('TWITTER_USERNAME', default='')  # TODO
+
 # Social Auth
 LOGIN_URL = 'gh_login'
+LOGIN_URL = 'twitter_login'
```

Comment:
can i still login with github too?

Reply:
I found it working even with these lines commented. Maybe it is useless?
![image](https://user-images.githubusercontent.com/16816538/56779351-34ba5100-680d-11e9-90a9-6ffcf8a41130.png)